packages/astro/test/fixtures/lit-element/src/pages/index.astro

```diff
@@ -1,18 +1,18 @@
 ---
-import '../components/my-element.js';
+import {MyElement} from '../components/my-element.js';
 ---
 
 <html>
 <head>
   <title>LitElements</title>
 </head>
 <body>
-  <my-element
+  <MyElement
       foo="bar"
       str-attr={'initialized'}
       bool={false}
       obj={{data: 1}}
       reflectedStrProp={'initialized reflected'}>
-  </my-element>
+  </MyElement>
```

Comment:
Idea that may or may not be feasible: if we're already crawling the dependency graph to match components, could we have a plugin that adds `meta.astro.tagName` to files that export `tagName`?

Certainly not blocking but I do think this is a big change for Lit users.

Reply:
I looked into doing this. Problem is that it requires parsing the JS and getting an AST. Then attempting to account for every variation of JS that might be used. Ultimately it cannot cover every case that would work in the old runtime method. For example `export tagName = 'my-' + {a:'element'}['a']`. Not that people are likely to do that, but that's the burden you take on when you support dynamic JS based configuration.

It felt like the effort was not worth it given the low usage of this integration in general. If people have an issue with this change adding support for configuration like this would be easier to add:

```js
export default {
  integrations: [lit({
    elements: {
      'my-element': 'my-project/components/my-element.js'
    }
  })]
}
```